src/Common/tests/System/Net/Http/LoopbackServer.cs

```diff
@@ -355,6 +361,7 @@ public class Options
                 SslProtocols.Tls13 |
 #endif
                 SslProtocols.Tls | SslProtocols.Tls11 | SslProtocols.Tls12;
+            public List<SslApplicationProtocol> ApplicationProtocols;
```

Comment:
This should not be a public field.  This is against usual .NET design guidelines.

Instead, it should be a public property with a getter only to return a list.  The list should be initialized to an empty list.

Then consumers can simply do this:

```c#
Options options = new Options();
options.ApplicationProtocols.Add(SslApplicationProtocol.Http2);
```

Reply:
I had to roll this back as that broke tests outside of System.Net.Http built for netstandard(2.0)